### PR TITLE
Make use of Redux in React, useSelector, useDispatch, etc...

### PR DESCRIPTION
**Pull request by Javivy**
## In this milestone I implemented the following functions and enhancements:

  - [x] Made use of redux hooks such as `useSelector, useDispatch`
  - [x] Made use of the redux store in React
  - [x] Connected React with redux
  - [x] Made use of redux reducers and actions in React

### CODE REVIEW

File: src/components/Categories.js [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
+import { checkStatus } from '../redux/categories/categories.js';
+
+const Categories = () => {
+  const dispatch = useDispatch();
+
+  return (
     <div className="categories-container">
-      <button>Check Status</button>
+      <button onClick={() => {
+        console.log(dispatch(checkStatus()));
```
Comment (elisha2kyakpo1) [inline src/components/Categories.js]: - [ ] Kindly remove the console.log in the code, we only do that for testing purposes, please check the Javascript requirements for this project for reference.

File: src/components/DisplayBooks.js [hunk 2]
```diff
@@ -1,10 +1,16 @@
 /* eslint-disable no-unused-vars */
+import { useSelector } from 'react-redux';
 import Book from './Book.js';
 
-const DisplayBooks = () => (
-    <div className="books-list">
-      <Book title='The little prince' author='Antoine de Saint-Exupéry'/>
+const DisplayBooks = () => {
+  const myState = useSelector((store) => store);
+  console.log(myState.books);
```
Comment (elisha2kyakpo1) [inline src/components/DisplayBooks.js]: - [ ] Let's remove the console log function here as well.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Javivy, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The PR title looks good and descriptive 👍
- The readme file looks professional as well 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- The summary description can be improved a bit, you can state e.g in point form what you have done.
- It is advisable to commit often, this way it's even easy for others to follow what you did at every commit.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cwaku — APPROVED** [review]
Hi @Javivy 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. The previous reviewer simply missed some of them, and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**